omxd.c

```diff
@@ -58,10 +96,24 @@ int main(int argc, char *argv[])
 	return 0;
 }
 
-/* Fork, umask, SID, chdir, close, logfile, FIFO */
+/* Open the FIFO at /var/run/omxctl */
+static int open_fifo(void)
```

Comment:
Wrong name: it should be `create_fifo`. Opening the fifo drives the main loop, therefore this name is especially confusing. Or just leave the code in `main`, which will become much smaller anyway after removing `getopt`. :-)